packages/components/src/components/button-group/button-group-styles.ts

```diff
@@ -47,13 +27,10 @@ export const getComponentCss = (direction: BreakpointCustomizable<ButtonGroupDir
       },
       div: {
         display: 'flex',
-        marginTop: `-${spacingStaticMedium}`,
+        marginTop: `${spacingFluidSmall}`,
```

Comment:
String interpolation is useless.
I think something is wrong now, because this adds a `margin-top` to the component where there wasn't any in the past.

Reply:
In the past the margin was removed from the button group wrapper and added to the button itself. This also increases the distance between all buttons when direction="row" is selected. I didn't see any need for the margin to be on the buttons themselves, so I removed the margin from the buttons and set it to the button group.

The spacing is bigger now because of using the fluid spacing (discussed this with johannes on friday)

Reply:
I also don't think that marginTop is needed at all since, everything is now handled with flex gap.